.circleci/config.yml

```diff
@@ -219,6 +219,8 @@ jobs:
           POSTGRES_HOST_AUTH_METHOD: trust
     parallelism: 5
     working_directory: ~/pix/high-level-tests/e2e
+    environment:
+      INTERVAL_CHECK_APPLICATION_IS_UP_SECOND: 10
```

Comment:
Est-ce qu'on ne peut pas utiliser une variable de CircleCI pour ne pas devoir changer ce fichier si on souhaite modifier la durée de l'intervalle ?

Je suis pas fan du nom de la variable mais je n'ai pas d'inspiration pour un meilleur nom.

Reply:
Je peux faire ça !
https://app.circleci.com/settings/project/github/1024pix/pix/environment-variables?return-to=https%3A%2F%2Fcircleci.com%2Fdashboard

Mais ça demande de cliquer pour la trouver.. OK, 2 pouces, je fais ç a!

Reply:
Done !
J'ai renommé la variable pour que dans la liste des variables on sache à quoi ça sert